test/postman/osmt_apitest.sh

```diff
@@ -13,31 +13,24 @@ function setup_environment() {
 
```

Comment:
Based on the sketch in the JIRA ticket, this script should be named `run_api_tests.sh` and do only these things:

- npm install --save-dev
- get bearer token from Okta
- npm run test-newman (with any needed env vars)

Everything else should be removed from the script. It's not this script's job to check for software or install software.